internal/pkg/cli/pipeline_init.go

```diff
@@ -192,6 +201,49 @@ func (o *initPipelineOpts) Ask() error {
 	return nil
 }
 
+func (o *initPipelineOpts) validateDuplicatePipeline() error {
+	// make sure pipeline isn't already deployed
+	fullName := fmt.Sprintf(fmtPipelineName, o.appName, o.name)
```

Comment:
I wonder if we should check the existence of both
1. pipeline-app-pipelineName; and
2. pipelineName?

Reply:
This is a great idea! Should help avoid confusion between legacy and new pipelines.